moped-editor/src/views/dashboard/DashboardStatusModal.js

```diff
@@ -58,8 +58,8 @@ const DashboardStatusModal = ({
         fullWidth
         maxWidth={"md"}
       >
-        <DialogTitle className={classes.dialogTitle}>
-          <h3>{`Status update - ${projectName}`}</h3>
+        <DialogTitle className={classes.dialogTitle} variant="h4">
+          {`Status update - ${projectName}`}
```

Comment:
all these places that had `<h3>` or `<h2>` or whatever i removed bc i assume we want there to be complete unity across all the DialogTitles?

Reply:
yes—perfect! this squashes console errors, too :)

would you also remove the custom styles/classes anywhere they exist?